pkgs/by-name/li/libtrawldb/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  cmake,
+  json-glib,
+  gettext,
+  gobject-introspection,
+  intltool,
+  gtk3,
+  tinysparql,
+  vala,
+  libgee,
+  gtk-layer-shell,
+  glib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libtrawldb";
+  version = "0.1-3";
+
+  src = fetchFromGitHub {
+    owner = "regolith-linux";
+    repo = "libtrawldb";
+    rev = "v${version}";
+    hash = "sha256-MqneoDFemBFf4PdX/hEfOB6ml4Rtjb1HF4R5QXfwiYE=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    cmake
+    ninja
+    json-glib
+    gettext
+    gobject-introspection
+    intltool
+    gtk3
+    tinysparql
+    vala
+    libgee
+    gtk-layer-shell
+    glib
+  ];
+
+  installPhase = ''
+    mkdir -p $out
+    meson install --destdir $out
+    ninja
+    mkdir -p $out/lib
+    cp -r $out/usr/lib $out
+    cp -r $out/nix/store/*/* $out
+    rm -rf $out/nix
+  '';
```

Comment:
this is probably not necessary and is actually actively harmful. You can probably remove this entirely.